<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Temperatura</class>
 <widget class="QDialog" name="Temperatura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>198</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Temperatura</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/ícono</normaloff>:/ícono</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.903409 rgba(235, 194, 61, 255), stop:1 rgba(0, 0, 0, 255));
font: 10pt &quot;Segoe Print&quot;;</string>
     </property>
     <property name="title">
      <string>Centigrados</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLCDNumber" name="lcdCent">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(110, 129, 255);
alternate-background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialCent">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="minimum">
         <number>-275</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCent">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
font: 10pt &quot;Segoe Print&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.903409 rgba(235, 194, 61, 255), stop:1 rgba(0, 0, 0, 255));</string>
     </property>
     <property name="title">
      <string>Fahrenheit</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLCDNumber" name="lcdFahr">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(69, 139, 208);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialFahr">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="minimum">
         <number>-460</number>
        </property>
        <property name="maximum">
         <number>210</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblFahr">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Segoe Print&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.903409 rgba(235, 194, 61, 255), stop:1 rgba(0, 0, 0, 255));</string>
     </property>
     <property name="title">
      <string>Kelvin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLCDNumber" name="lcdKel">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(69, 139, 208);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dialKel">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="maximum">
         <number>374</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblKel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
